Schedule reports docs update (#3860)
SumoLogic/sumologic-documentation: blog-service/2024-05-31-dashboards.md

---
title: Multiple Scheduled Reports (Dashboards)
image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - search
  - scheduled_reports
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

> 

Reviewer: @JV0812, you'll need to change the date on the release note, of course. Kevin Keech said in his [presentation](https://sites.google.com/sumologic.com/pd-global-home/products/explore-new-products#h.sek1vyg6k6rx) that it will release "sometime in the next two weeks".

> We are excited to introduce the **Multiple Scheduled Reports** feature to our dashboards, this powerful new capability allows you to configure multiple reports against a dashboard, providing enhanced flexibility and control over your reporting needs. Here are some of the key features that this feature offers:

Reviewer 2: ```suggestion
We are excited to introduce the **Multiple Scheduled Reports** feature to Sumo Logic dashboards. This powerful new capability allows you to configure multiple reports against a dashboard, providing enhanced flexibility and control over your reporting needs. Key features include:
```

- **Multiple Configurations**. Set up several reports from the same dashboard, each with its unique settings.
- **Custom Variable Values**. Assign different variable values to each report, tailoring the data to meet specific reporting needs.
- **Custom Time Ranges**. Define distinct time ranges for each report, ensuring you capture the data most relevant to your analysis.
- **Run Frequencies**. Schedule reports to run at varying frequencies, from daily snapshots to monthly summaries.

> - **Email Addresses**. Specify different email addresses for each report, ensuring the right information reaches the right stakeholders.

Reviewer 3: Suggestion:
different email addresses for each report-> one or more email addresses for each report

> - **Email Addresses**. Specify different email addresses for each report, ensuring the right information reaches the right stakeholders.

Author: ```suggestion
- **Email Addresses**. Specify one or more email addresses for each report, ensuring the right information reaches the right stakeholders.
```

> - **Multiple Configurations**. Set up several reports from the same dashboard, each with its unique settings.
> - **Custom Variable Values**. Assign different variable values to each report, tailoring the data to meet specific reporting needs.
> - **Custom Time Ranges**. Define distinct time ranges for each report, ensuring you capture the data most relevant to your analysis.
> - **Run Frequencies**. Schedule reports to run at varying frequencies, from daily snapshots to monthly summaries.
> - **Email Addresses**. Specify one or more email addresses for each report, ensuring the right information reaches the right stakeholders.

Reviewer: ```suggestion
- **Multiple configurations**. Set up several reports from the same dashboard, each with its unique settings.
- **Custom variable values**. Assign different variable values to each report, tailoring the data to meet specific reporting needs.
- **Custom time ranges**. Define distinct time ranges for each report, ensuring you capture the data most relevant to your analysis.
- **Run rrequencies**. Schedule reports to run at varying frequencies, from daily snapshots to monthly summaries.
- **Email addresses**. Specify one or more email addresses for each report, ensuring the right information reaches the right stakeholders.
```
